src/utils.js

```diff
@@ -559,6 +559,22 @@ export const updateDirectoryFile = (folderContent, folderOrder) => {
   return Base64.encode(yaml.safeDump(decodedContent))
 }
 
+export const getNavFolderDropdownFromFolderOrder = (folderOrder) => {
+  return folderOrder.reduce((acc, curr) => {
+    const pathArr = curr.split('/') // sample paths: "prize-sponsor.md", "prize-jury/nominating-committee.md"
+
+    if (pathArr.length === 1) {
+      acc.push(deslugifyDirectory(curr.split('.')[0])) // remove file extension
+    }
+
+    if (pathArr.length === 2 && pathArr[0] !== acc[acc.length - 1]) {
+      acc.push(deslugifyDirectory(pathArr[0]))
```

Comment:
Should check if `deslugifyDirectory(pathArr[0])` already exists or use Set()

Reply:
Good catch, fixed with [5a34a49](https://github.com/isomerpages/isomercms-frontend/pull/362/commits/5a34a49627545011d7104cfa151e6c448e96dd15)